src/Algebra/Graph/Bipartite/Undirected/AdjacencyMap.hs

```diff
@@ -50,11 +57,12 @@ import Control.Monad
 import Control.Monad.Trans.Maybe
 import Control.Monad.State
 import Data.Either
-import Data.Foldable
+import Data.Foldable (asum)
```

Comment:
For history: we only use `asum` from `Data.Foldable`, and `Data.Foldable.toList` conflicts with `GHC.Exts.toList`. Might have been as well

```
import Data.Foldable hiding toList
```